test/crawlers/cheerio_crawler.test.js

```diff
@@ -670,61 +643,94 @@ describe('CheerioCrawler', () => {
 
             await crawler.run();
 
-            expect(proxies[0]).toEqual(proxyConfiguration.getInfo(sessions[0].id));
-            expect(proxies[1]).toEqual(proxyConfiguration.getInfo(sessions[1].id));
-            expect(proxies[2]).toEqual(proxyConfiguration.getInfo(sessions[2].id));
-            expect(proxies[3]).toEqual(proxyConfiguration.getInfo(sessions[3].id));
+            expect(proxies[0]).toEqual(proxyConfiguration.newProxyInfo(sessions[0].id));
+            expect(proxies[1]).toEqual(proxyConfiguration.newProxyInfo(sessions[1].id));
+            expect(proxies[2]).toEqual(proxyConfiguration.newProxyInfo(sessions[2].id));
+            expect(proxies[3]).toEqual(proxyConfiguration.newProxyInfo(sessions[3].id));
 
             delete process.env[ENV_VARS.PROXY_PASSWORD];
             stub.restore();
         });
 
-        describe('throws', () => {
-            /* eslint-disable no-new */
-            beforeEach(() => {
-                log.setLevel(log.LEVELS.OFF);
+        test('should work with proxyUrls configuration', async () => {
+            process.env[ENV_VARS.PROXY_PASSWORD] = 'abc123';
+            const proxies = [];
+            const status = { connected: true };
+            const fakeCall = async () => {
+                return { body: status };
+            };
+
+            const stub = sinon.stub(utilsRequest, 'requestAsBrowser').callsFake(fakeCall);
```

Comment:
This shouldn't be needed when using `proxyUrls`. It could obfuscate a problem.

Applies to a lot of tests.

Reply:
You are totally right, I did not realize that.